Makefile

```diff
@@ -31,11 +31,13 @@ IMAGE_REPO :=quay.io/microshift/microshift
 IMAGE_REPO_AIO :=quay.io/microshift/microshift-aio
 OUTPUT_DIR :=_output
 RPM_BUILD_DIR :=packaging/rpm/_rpmbuild
+ISO_DIR :=scripts/image-builder/_builds
 CROSS_BUILD_BINDIR :=$(OUTPUT_DIR)/bin
 FROM_SOURCE :=false
 CTR_CMD :=$(or $(shell which podman 2>/dev/null), $(shell which docker 2>/dev/null))
 ARCH :=$(shell uname -m |sed -e "s/x86_64/amd64/" |sed -e "s/aarch64/arm64/")
 IPTABLES :=nft
+PULLSECRET :=~/pullsecret
```

Comment:
I think we should be consistent with the documentation where pull secret files are placed in ~/pull-secret.txt

See [Install MicroShift on RHEL for Edge](https://github.com/openshift/microshift/blob/main/docs/rhel4edge_iso.md#prerequisites)